### DISCUSSION TITLE
Masowe podnoszenie wersji TecDoc

### DISCUSSION THREAD

**Discussion by bugproof** [allegro/allegro-api#10539] [Suggestions]
Allegro samo zakańcza oferty co jest poza naszym wpływem. Podnosi wersje TecDoc w swoim katalogu i należy od nowa łączyć ofertę z katalogiem - nie dzieje się to automatycznie.

https://github.com/allegro/allegro-api/issues/10537

Wtedy w przypadku posiadania np. 100.000 takich ofert należy wywołać PATCH dla każdej z nich.

Jest to raczej coś co Allegro mogłoby wspierać bardziej natywnie

**Comment by PrzemyslawLukanowski** [comment]
Dziękuję za sugestię. Jeśli zdecydujemy się wprowadzić takie rozwiązanie, poinformujemy w wątku.
